CHANGELOG.md

```diff
@@ -6,6 +6,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [2.4.2] - 2020-08-26
+### Updated
```

Comment:
👍🏾  for keeping the CHANGELOG up to date. It'll be better to link to the issue using Markdown format. In GitHub, we can link an issue to a PR and hence there will be that chain of details (CHANGELOG -> issue -> PR) for future reference. This can also be requested as part of pull request submission - i.e. each PR need to contain an update to the CHANGELOG describing what has changed. That would make it easy to maintain the changelog.

E.g. entry:

` Updated README. [#1](https://github.com/rokwire/safer-illinois-app/issues/1)`